src/systems/filecoin_vm/sysactors/reward_actor.go

```diff
@@ -70,7 +74,8 @@ func (a *RewardActorCode_I) Constructor(rt vmr.Runtime) InvocOutput {
 	panic("TODO")
 }
 
-func (a *RewardActorCode_I) MintReward(rt vmr.Runtime) {
+func (a *RewardActorCode_I) AwardBlockReward(rt vmr.Runtime, miner addr.Address, penalty actor.TokenAmount) {
```

Comment:
should add a placeholder to handle penalty larger than blockReward (as deleted from `vm_interpreter.go` L296)